client/src/app/gateways/presenter/server-time-presenter.service.ts

```diff
@@ -66,13 +56,20 @@ export class ServerTimePresenterService {
      */
     private async refreshServertime(): Promise<void> {
         // servertime is the time in seconds.
-        const servertimeResponse = await this.presenter.call<ServerTimeResponse>(Presenter.SERVERTIME);
-        if (typeof servertimeResponse?.server_time !== `number`) {
```

Comment:
This could be the cause of https://github.com/OpenSlides/openslides-client/issues/3503 because the server time is used for the countdowns and NaN is actually a number and would therefore bypass this check.